### PR TITLE
New health bar

### PR DESCRIPTION
**Pull request by whitew1994WW**
New cool plunger

### CODE REVIEW

**Comment by wattsit** [comment]
image
in `HUD.gd` returns null.

I think you need to include the changes to the `main_game.tscn`

**Review by wattsit — CHANGES_REQUESTED** [review]
Needs the changes to `main_game.tscn` included in the pull.

**Review by kepler471 — APPROVED** [review]

File: Moonshot/HUD.gd [hunk 1]
```diff
@@ -8,14 +8,20 @@ var weapon_textures = {
...
 
+var plunger
+
+func _ready():
+	plunger = get_parent().get_node("GUI/PlungerPosition/Plunger")
 	
 func _process(delta):
-	var new_health_bar_size = (Utils.player_stats.health*154) / Utils.player_stats.max_health
-	var health_bar = get_parent().get_node("GUI/MarginContainer/VBoxContainer/VBoxContainer3/VBoxContainer2/HBoxContainer2/HealthBar")
+	var new_health_bar_size = (Utils.player_stats.health*152) / Utils.player_stats.max_health
```
Comment (Jamesbarford) [inline Moonshot/HUD.gd:17]: Can we make make the magic numbers constants?

**Review by Jamesbarford — CHANGES_REQUESTED** [review]

File: Moonshot/HUD.gd (hunk 1 above)
Comment (Jamesbarford) [reply Moonshot/HUD.gd:17]: For the next pr

**Pull request merged**